src/Request/AbstractWirecardRequest.php

```diff
@@ -37,7 +37,7 @@ function getEndpoint()
         return $this->endpoint;
     }
 
-    function setEndpoint($endpoint)
+    function setEndpoint(String $endpoint)
```

Comment:
That won't work in PHP < 7.0

Remove the "String" typehint and it will work. Also it would be good to add a testcase for that, as a testcase for the setter would have catched that error.